src/d3a/d3a_core/export.py

```diff
@@ -244,13 +259,14 @@ def _export_area_stats_csv_file(self, area: Area, directory: dir, balancing: boo
             area_name += "-balancing"
         data = self.export_data.generate_market_export_data(area, balancing)
         rows = data.rows()
-        if not rows:
+        if not rows and not is_first:
```

Comment:
Can you please explain, why you added `and not is_first`  here? Thanks!

Reply:
Without this additional check, it will return without adding a header in CSV.